### PR TITLE
[PRO-3017] Filtering on unassigned tasks

### PR DESCRIPTION
**Pull request by mikeverf**
#### Changed
- `user_id` filter on `tasks.list` should be nullable (Make filtering on unassigned tasks possible)

### CODE REVIEW

**Review by tiagobastos — DISMISSED** [review]

**Review by EOSullivanBerlin — DISMISSED** [review]

**Review by ArnaudWeyts — DISMISSED** [review]

**Review by luisaterrabrito — DISMISSED** [review]

**Comment by adognameddog** [comment]
:white_check_mark: Manually checked

**Review by adognameddog — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -3400,7 +3400,7 @@ Get a list of tasks.
     + Attributes (object)
         + filter (object, optional)
             + ids: `9c475de2-1e99-0328-bc19-271b27e921d8`,`6325b4e5-c6a6-060b-b010-b81892e82fa7` (array[string], optional)
-            + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional) - Returns Tasks that are assigned to this user or to a team to which this user belongs.
+            + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional, nullable) - Returns Tasks that are assigned to this user or to a team to which this user belongs. When passing `null`, returns tasks that are unassigned.
```
Comment (ketlane) [inline apiary.apib]: PP on the English comment: I would rephrase this as `When passing null, it returns tasks` instead of `When passing null, returns tasks`. Also, the beginning of the sentence is `Returns Tasks that are assigned to this user`; I don't see why we use a capital letter for tasks (`Returns Tasks`), I would use `Returns tasks` (I realize this isn't something introduced in this PR, but it grabbed my attention).

**Review by ketlane — DISMISSED** [review]

File: apiary.apib (hunk 1 above)
Comment (mikeverf) [reply apiary.apib]:
```suggestion
            + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional, nullable) - Returns tasks that are assigned to this user or to a team to which this user belongs. When passing `null`, it returns tasks that are unassigned.
```

**Review by tiagobastos — DISMISSED** [review]

**Review by luisaterrabrito — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by adognameddog — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Review by joaoinacio — APPROVED** [review]

**Review by ketlane — APPROVED** [review]
